### ISSUE TITLE
Incorrect (cached?) source string in webui

### ISSUE THREAD

**Issue by burner1024** [WeblateOrg/weblate#2104]
### Steps to reproduce
1. Generate new POT, merge into PO
2. Verify that repo is up to date in weblate
3. Flush redis cache. Restart both uwsgi and Nginx

### Actual behaviour
Incorrect old string is displayed in "[original](https://tra.bgforge.net/translate/infinity-engine/ascension/ru/?type=fuzzy&offset=4)" field. Says "FaerÃ»n" instead of "Faerûn". Here's the [PO file](https://github.com/BigWorldProject/Ascension/blob/fe7e74fdc4e31787974e38a510e97061e74bd24e/ascension/tra/po/russian.po#L5045).
I've flushed and restarted everything I could think of, still no dice.

### Expected behaviour
Correct string should appear in web ui, without any cache flushing or restarting services.

### Server configuration
```
 * Weblate weblate-2.17
 * Python 2.7.12
 * Django 1.11.7
 * six 1.11.0
 * social-auth-core 1.5.0
 * social-auth-app-django 2.0.0
 * django-appconf 1.0.2
 * Translate Toolkit 2.2.5
 * Whoosh 2.7.4
 * defusedxml 0.5.0
 * Git 2.7.4
 * Pillow (PIL) 1.1.7
 * dateutil 2.6.1
 * lxml 4.1.1
 * django-crispy-forms 1.7.0
 * compressor 2.2
 * djangorestframework 3.7.3
 * pytz 2017.3
 * pyuca N/A
 * python-bidi 0.4.0
 * pyLibravatar N/A
 * PyYAML 3.12
 * Mercurial 4.4.1
 * Database backends: django.db.backends.mysql
```

**Comment by nijel** [comment]
Does that get fixed on running `./manage.py loadpo infinity-engine/ascension`? If yes, that most likely the VCS update didn't complete for some reason. Check logs for errors.

If that doesn't help, try `./manage.py loadpo --force infinity-engine/ascension`.

I think there was a race condition making Weblate skip some updates in some corner cases, but I fail to find details right now. I think it has been fixed in 2.20, but I'm really not sure.

**Comment by burner1024** [comment]
Yes, first cmd fixed the issue. I'll try updating soon. Will report if it resurfaces.
